### ISSUE TITLE
infoUrl cannot be set by data-* attribute

### ISSUE THREAD

**Issue by neffets** [heiseonline/shariff#44]
In current Firefox, its not possible to add an own infoUrl per data-attribute

```javascript
$('.shariff').attr('data-infoUrl', 'http://www.berlin.de/wir-ueber-uns-be/agb/pp/20150101/#socialplugins');
```

The browser will add the data-attribute, BUT it will become a "data-infourl"  (all lowercase).

There are three ways to solve it:

* a) accept a "data-infourl" too, for overwriting the infoUrl parameter
* b) change the internal infoUrl option to "infourl" (all lowercase), or
* c) Give the possibility to give an whole options-json-array to _Shariff
  * as c1) later callable as global function _Shariff(this, {infoUrl: 'http://www...'})    
    see Issue #34     
  * or as c2) a new data attribute data-options:    
     data-options="{infoUrl: 'http://www...', theme: 'white'}"

**Comment by pmb0** [comment]
Hi!

First, jQuery does an automatic conversion of data key names. Example:

```js
> jQuery("body").data("foo-bar", "baz").data()
< Object {fooBar: "baz"}
```

So, if you want to set the info url, you would have to use something like `$("shariff").data("info-url", "blah")` (also notice `data()` instead of `attr()`). Since [`Object.observe()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe) is not supported by most browsers, Shariff can't detect changes made on the data list. And there is no `update()` method on the shariff object yet.

I just pushed a (not released) change, that renames `_Shariff` to `Shariff` and that exports the class to the global window object. I also pushed [an example](https://github.com/heiseonline/shariff/blob/master/demo/custom-initialization.html) that shows how to pass own options to the Shariff constructor method.

Summary:

a) use `data-info-url` instead of `data-infourl`/`data-infoUrl`
b) there is no reason to change it
c) Custom options could always be passed to the constructor. The change, that sets the `Shariff` class in the global window object, will be released soon.
